src/chat/language_channel_config.py

```
@@ -12,34 +14,18 @@ def __init__(self, name, icon, checked):
-        self._form = formc()
-        self._base = basec(self._parent_widget)
-        self._form.setupUi(self._base)
-        self._form.endDialogBox.accepted.connect(self._on_accepted)
-        self._form.endDialogBox.rejected.connect(self._on_rejected)
-
-    def _setup_model(self):
         self._model = CheckableStringListModel()
-        self._form.channelListView.setModel(self._model)
 
-    def _load_data(self):
+    def model(self) -> CheckableStringListModel:
```

Ruff (0.15.6)

[error] Undefined name `CheckableStringListModel`

(F821)